pkg/dd9a3433-a992-41f4-b8d1-086095a9b54e/package.lp

```diff
@@ -0,0 +1,72 @@
+(librepcb_package dd9a3433-a992-41f4-b8d1-086095a9b54e
+ (name "KEYSTONE_2468")
+ (description "Battery holder (open) for 2 cells AAA with through-hole pins.")
+ (keywords "keystone,battery,holder,cell,aaa")
+ (author "Jessy")
+ (version "0.1")
+ (created 2019-07-29T20:54:43Z)
+ (deprecated false)
+ (category 1c4e8cee-9c15-4dc4-997c-3bee75a0c1c4)
+ (pad 61856d8f-3a7c-4219-b680-451922719511 (name "+"))
+ (pad d6432d99-3eed-4bdc-997a-f96fd80ba685 (name "-"))
+ (footprint 49510f62-6bc1-4f5c-96e3-22da1b110766
+  (name "default")
+  (description "")
+  (pad d6432d99-3eed-4bdc-997a-f96fd80ba685 (side tht) (shape round)
+   (position -6.35 20.32) (rotation 90.0) (size 3.6 1.8) (drill 1.02)
+  )
+  (pad 61856d8f-3a7c-4219-b680-451922719511 (side tht) (shape round)
+   (position 6.35 20.32) (rotation 90.0) (size 3.6 1.8) (drill 1.02)
+  )
+  (polygon 326a1398-0a86-463f-9c7f-efb87600d382 (layer top_placement)
+   (width 0.2) (fill false) (grab_area true)
+   (vertex (position -12.395 26.59) (angle 0.0))
+   (vertex (position 12.395 26.59) (angle 0.0))
+   (vertex (position 12.395 -26.59) (angle 0.0))
+   (vertex (position -12.395 -26.59) (angle 0.0))
+   (vertex (position -12.395 26.59) (angle 0.0))
```

Comment:
The `12.395` and the `26.59` seem a bit odd. Shouldn't it be 12.4 and 26.6? From the datasheet at https://www.keyelco.com/userAssets/file/M65p28.pdf it looks like the device is 53.0 x 24.6mm. So 53.0 / 2 + 0.1 (half the trace width) = 26.6.

Reply:
The "Drawing" URL is more exact: http://www.keyelco.com/product-pdf.cfm?p=1033

52.98 / 2 + 0.1 = 26.59 :slightly_smiling_face: